I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 112, in script
    call screen game_over_screen
  File "renpy/common/000statements.rpy", line 569, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "game/screens.rpy", line 1521, in execute
    screen game_over_screen:
  File "game/screens.rpy", line 1521, in execute
    screen game_over_screen:
  File "game/screens.rpy", line 1523, in execute
    frame:
  File "game/screens.rpy", line 1524, in execute
    vbox:
  File "game/screens.rpy", line 1526, in execute
    hbox:
  File "game/screens.rpy", line 1530, in execute
    textbutton "Sorry" xalign 0.33 yalign 0.5 action MainMenu(confirm=false)
  File "game/screens.rpy", line 1530, in keywords
    textbutton "Sorry" xalign 0.33 yalign 0.5 action MainMenu(confirm=false)
  File "game/screens.rpy", line 1530, in <module>
    textbutton "Sorry" xalign 0.33 yalign 0.5 action MainMenu(confirm=false)
NameError: name 'false' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 112, in script
    call screen game_over_screen
  File "renpy/ast.py", line 2015, in execute
    self.call("execute")
  File "renpy/ast.py", line 2003, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "renpy/statements.py", line 278, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 569, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "renpy/exports.py", line 3136, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "renpy/ui.py", line 298, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "renpy/display/core.py", line 3325, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs)
  File "renpy/display/core.py", line 3737, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "renpy/display/core.py", line 568, in visit_all
    d.visit_all(callback, seen)
  File "renpy/display/core.py", line 568, in visit_all
    d.visit_all(callback, seen)
  File "renpy/display/core.py", line 568, in visit_all
    d.visit_all(callback, seen)
  File "renpy/display/screen.py", line 436, in visit_all
    callback(self)
  File "renpy/display/core.py", line 3737, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "renpy/display/screen.py", line 447, in per_interact
    self.update()
  File "renpy/display/screen.py", line 637, in update
    self.screen.function(**self.scope)
  File "game/screens.rpy", line 1521, in execute
    screen game_over_screen:
  File "game/screens.rpy", line 1521, in execute
    screen game_over_screen:
  File "game/screens.rpy", line 1523, in execute
    frame:
  File "game/screens.rpy", line 1524, in execute
    vbox:
  File "game/screens.rpy", line 1526, in execute
    hbox:
  File "game/screens.rpy", line 1530, in execute
    textbutton "Sorry" xalign 0.33 yalign 0.5 action MainMenu(confirm=false)
  File "game/screens.rpy", line 1530, in keywords
    textbutton "Sorry" xalign 0.33 yalign 0.5 action MainMenu(confirm=false)
  File "game/screens.rpy", line 1530, in <module>
    textbutton "Sorry" xalign 0.33 yalign 0.5 action MainMenu(confirm=false)
NameError: name 'false' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
GoopChild 1.0
Wed Feb  2 13:50:17 2022
